ui/packages/shared/profile/src/ProfileSource.tsx

```diff
@@ -239,13 +255,13 @@ export class MergedProfileSource implements ProfileSource {
   }
 
   ProfileType(): ProfileType {
-    return ProfileType.fromString(Query.parse(this.query).profileName());
+    return ProfileType.fromString(Query.parse(this.query()).profileName());
   }
 
   Describe(): JSX.Element {
     return (
       <a>
-        Merge of &quot;{this.query}&quot; from {formatDate(this.mergeFrom, timeFormat)} to{' '}
+        Merge of &quot;{this.query()}&quot; from {formatDate(this.mergeFrom, timeFormat)} to{' '}
```

Comment:
can be written using template literals (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

`Merge of "${this.query()}" from ${formatDate(this.mergeFrom, timeFormat)} to...`

Reply:
using `